README.md

```diff
@@ -1,4 +1,6 @@
 easy-datacite
 =============
 
+[![Build Status](https://travis-ci.org/DANS-KNAW/easy-datacite.svg?branch=master)](https://travis-ci.org/DANS-KNAW/easy-datacite)
```

Comment:
Nope, this only applies to the public projects that have Travis support. And even then the URL is very specific...